### ISSUE TITLE
Bug: Mod is listed as client side on modrinth but gives mod mismatch when missing on server.

### ISSUE THREAD

**Issue by evelynisthebest** [TeamMidnightDust/MidnightControls#303]
**Describe the bug**
The Mod is listed on Modrinth as a Client side mod but when it is installed on only the client server gives a mod mismatch error on connect. For NeoForge 21.1.31

**To Reproduce**
1. Install MidnightControls on client side only
2. Do NOT install Midnight Controls on Server
3. Attempt to connect to server
4. Mod Mismatch Error

**Expected behavior**
Either the mod should be listed as Client and Server Required or there should not be a mismatch when connecting.

**Screenshots**
![image](https://github.com/user-attachments/assets/e6f46d38-a3d8-492d-9a35-a65152af4332)

**Desktop (please complete the following information):**
 - OS: Client Windows, Server Linux
 - Minecraft 1.21.1
 - Modloader NeoForge 21.1.31
 - Fabric/Quilt Libraries n/a
 - Mods ATM10
 - Version 41

**Additional context**
In case of a crash, please provide the crash log.

**Comment by alzacastar** [comment]
+1

I have the same issue.  If the mod is installed on the client, but not the server it causes a mismatch error. If you happen to install it on both sides it does appear to work correctly (at least with some initial testing). 
But if it is missing from either client or server it causes the mismatch error. 

This appears only to be happening with the NeoForge version, as it is working as intended on the Fabric version of MidnightControls (where the mod is only installed on client side)

OS: Client Windows, Server Windows
Minecraft 1.21.1
Modloader NeoForge 21.1.30
Mod Pack: New Game (v9.1.0)
MidnightControls Version: 1.10.0-beta.1-neoforge

**Comment by Motschen** [comment]
Yeah, NeoForge support is still quite experimental, that's why it's still in beta.
This issue will be fixed in the full release.
